templates/partials/receipts-tab.html

```diff
@@ -5,9 +5,17 @@
 <section class="page-section" id="section-2" role="tabpanel" aria-hidden="true" aria-labelledby="section-2-heading">
   <div class="container">
     <div class="page-section__intro">
-      <h2 class="section-heading" id="section-2-heading">Analyze Receipts {% include 'partials/cycle-select.html' %}  </h2>
-      <div class="toggles section__intro__left">
-        <span class="label">Compare by:</span>
+      <div class="section__intro__left">
```

Comment:
The "left" part sounds more like a modifier, so should probably be `--left`. If there are too many places this would have to change, I don't care that much.